client/src/components/Collections/PairedOrUnpairedListCollectionCreator.vue

```diff
@@ -0,0 +1,798 @@
+<script setup lang="ts">
```

Comment:
I seem to be able to feed a nested list into this builder, and it somehow picks a dataset out of that nested list, except that dataset is not actually part of the input dataset collection ? This seemed really weird.

Reply:
What do you mean feed a nested list into the builder? Like using the history menu and history element selection?

Reply:
Yes. I'll create a separate bug report, doesn't seem necessary to block the PR on that.